e2e/tests/init_test.go

```diff
@@ -418,7 +418,13 @@ func (s *suite) TheApplicationSendsTheSupplierASuccessfulRequestForServiceWithPa
 	// that allows any application to send a relay in LocalNet and our E2E Tests.
 	require.Equal(s, "app1", appName, "TODO_HACK: The LocalNet AppGateServer is self_signing and only supports app1.")
 
-	res, err := s.pocketd.RunCurlWithRetry(appGateServerUrl, serviceId, path, requestData, 5)
+	method := "POST"
+	// If requestData is empty, assume a GET request
+	if requestData == "" {
+		method = "GET"
+	}
```

Comment:
Same as below, can you add a println statement in case data is provided but we're ignoring it?

Reply:
I don't this it applies to this case. We are using `requestData` to infer the `method` here.